### ISSUE TITLE
Weak anal sphincter tone

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9859]
**Preferred term label:**
Weak anal sphincter tone

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
spinal epidural abscess

**Your nano-attribution (ORCID)**
https://www.ncbi.nlm.nih.gov/medgen/1783328

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
Weak anal sphincter tone
HP:0034917
